tests/boards/espressif_esp32/cache_coex/src/cache_coex.c

```diff
@@ -11,6 +11,7 @@
 #include <zephyr/drivers/flash.h>
 #include <zephyr/random/random.h>
 #include <soc/soc_memory_layout.h>
+#include <zephyr/multi_heap/shared_multi_heap.h>
 
 /* definitions used in Flash & RAM operations */
 #define SPIRAM_ALLOC_SIZE               (24 * 1024)
```

Comment:
Should this be retrieved from dts instead?

Reply:
This is just the buffer size for the test. It is there since the test creation. I don't think it should be a dts related prop.